.github/workflows/package.yml

```
@@ -0,0 +1,97 @@
+name: package
+on: [push, pull_request]
+jobs:
+  linux_wheels:
```

```suggestion
  linux-wheels:
```

(Here and elsewhere, for consistency.)

---

Fixed